package.json

```diff
@@ -134,7 +135,7 @@
   },
   "lint-staged": {
     "*.(js|ts|tsx)": [
-      "eslint --ext '.js,.ts,.tsx' src/ --ignore-pattern src/reanimated1 --ignore-pattern react-native-reanimated.d.ts --ignore-pattern docs",
+      "eslint --ext '.js,.ts,.tsx' src/ --ignore-pattern src/reanimated1 --ignore-pattern react-native-reanimated.d.ts --ignore-pattern docs --ignore-pattern plugin",
```

Comment:
Compiling with `"module" : "commonjs"` results with code unaccepted by linter (TSC gives snake case). Simpliest override for now is to ignore `/plugin` for now.